modules/cnf-topology-aware-lifecycle-manager-backup-feature.adoc

```diff
@@ -0,0 +1,160 @@
+        - device: /dev/sda
+        partitions:
+        - mount_point: /var/recovery
+            size: 51200
+            start: 800000
+----
+====
+
+.Procedure
+
+. Save the contents of the `ClusterGroupUpgrade` CR with the `backup` field set to `true` in the `clustergroupupgrades-group-du.yaml` file:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: du-upgrade-4918
+  namespace: ztp-group-du-sno
+spec:
+  preCaching: true
+  deleteObjectsOnCompletion: true
+  backup: true
+  clusters:
+  - cnfdb1
+  - cnfdb2
+  enable: false
+  managedPolicies:
+  - du-upgrade-platform-upgrade
+  remediationStrategy:
+    maxConcurrency: 2
+    timeout: 240
+----
+
+. To start the update, apply the `ClusterGroupUpgrade` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f clustergroupupgrades-group-du.yaml
+----
+
+.Verification
+
+. Check the status of the upgrade in the hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get cgu -n ztp-group-du-sno du-upgrade-4918 -o jsonpath='{.status}'
+----
++
+.Example output
++
+[source,json]
+----
+{
+    "backup": {
+        "clusters": [
+            "test-spoke2",
+            "test-spoke1"
+    ],
+    "status": {
+        "test-spoke1": "Succeeded",
+        "test-spoke2": "Succeeded"
+    }
+},
+"computedMaxConcurrency": 1,
+"conditions": [
+    {
+        "lastTransitionTime": "2022-04-05T10:37:19Z",
+        "message": "Backup is completed",
+        "reason": "BackupCompleted",
+        "status": "True",
+        "type": "BackupDone"
+    }
+],
+"precaching": {
+    "spec": {}
+},
+"status": {}
+----
+
+. Check the status of the upgrade by running the following command:
++
+[source,terminal]
+----
+$ oc ostree admin status
+----
+.Example outputs
++
+[source,terminal]
+----
+[root@lab-test-spoke2-node-0 core]# ostree admin status
+* rhcos c038a8f08458bbed83a77ece033ad3c55597e3f64edad66ea12fda18cbdceaf9.0
+    Version: 49.84.202202230006-0
+    Pinned: yes <1>
+    origin refspec: c038a8f08458bbed83a77ece033ad3c55597e3f64edad66ea12fda18cbdceaf9
+----
+<1> The current deployment is pinned. A rollback is not necessary.
++
+[source,terminal]
+----
+[root@lab-test-spoke2-node-0 core]# ostree admin status
+* rhcos f750ff26f2d5550930ccbe17af61af47daafc8018cd9944f2a3a6269af26b0fa.0
+    Version: 410.84.202204050541-0
+    origin refspec: f750ff26f2d5550930ccbe17af61af47daafc8018cd9944f2a3a6269af26b0fa
+rhcos ad8f159f9dc4ea7e773fd9604c9a16be0fe9b266ae800ac8470f63abc39b52ca.0 (rollback)
+    Version: 410.84.202203290245-0
+    Pinned: yes <1>
+    origin refspec: ad8f159f9dc4ea7e773fd9604c9a16be0fe9b266ae800ac8470f63abc39b52ca
+----
+<1> An upgraded deployment has been pinned.
++
+To trigger a rollback of pinned deployment, `rhcos ad8f159f9dc4ea7e773fd9604c9a16be0fe9b266ae800ac8470f63abc39b52ca.0`, run the rollback with the following command:
++
+[source,terminal]
+----
+$ rpm-ostree rollback -r
+----
```

Comment:
Here you have only described the first part. `rpm-ostree rollback -r` only restore the OS functionality, it does not restore the OCP state. You can follow here to include necessary information: [https://github.com/openshift-kni/cluster-group-upgrades-operator/tree/main/docs/backup-recovery#recovery-utility](url)

Reply:
added new information